runtime/v2/runner/src/test/resources/com/walmartlabs/concord/runtime/v2/runner/callWithErrorBlock/concord.yml

```diff
@@ -0,0 +1,13 @@
+flows:
+  default:
+    - call: createMyVar
+      out:
+        - myVar
+      error:
+        - log: "caught an error"
+    - log: "myVar should exist: ${myVar}"  # doesn't work
```

Comment:
The comment is no longer relevant, I think?

Reply:
ouch. fixed.